src/code/load.c

```diff
@@ -17,14 +17,17 @@ s32 Overlay_Load(uintptr_t vRomStart, uintptr_t vRomEnd, void* vRamStart, void*
 
     if (gOverlayLogSeverity >= 3) {
         // "DMA transfer of TEXT, DATA, RODATA + rel (%08x-%08x)"
-        osSyncPrintf("TEXT,DATA,RODATA+relをＤＭＡ転送します(%08x-%08x)\n", allocatedVRamAddr, end);
+        osSyncPrintf("TEXT,DATA,RODATA+relをＤＭＡ転送します(%08x-%08x)\n", allocatedRamAddr, end);
     }
 
-    DmaMgr_RequestSync(allocatedVRamAddr, vRomStart, size);
+    // DMA the overlay, wait until transfer completes
+    DmaMgr_RequestSync(allocatedRamAddr, vromStart, size);
 
+    // The overlay file is expected to contain a 32-bit offset from the end of the file to the start of the
+    // relocation section.
     ovlOffset = ((s32*)end)[-1];
-
     ovl = (OverlayRelocationSection*)(end - ovlOffset);
```

Comment:
Perhaps rename these `relocsOffset` and `ovlRelocs` respectively? Seems a little clearer, and also mirrors how you rename `ovlInfo` to `ovlRelocs` in `Overlay_Relocate`.

Reply:
Changed to `relocSectionOffset` and `ovlRelocs`